### PR TITLE
Hardware: Add support for ZeroOne X6

### PR DESCRIPTION
**Pull request by ZeroOne-Aero**
Dear Professors,I just added the support for ZeroOne X6，we are looking forward to your review and merger.

### CODE REVIEW

File: src/Comms/USBBoardInfo.json [hunk 1]
```diff
@@ -63,9 +63,12 @@
         { "vendorID": 8352, "productID": 16733,     "boardClass": "OpenPilot",  "name": "OpenPilot CC3D" },
         { "vendorID": 8352, "productID": 16734,     "boardClass": "OpenPilot",  "name": "OpenPilot Revolution" },
         { "vendorID": 8352, "productID": 16848,     "boardClass": "OpenPilot",  "name": "Taulabs Sparky2" }
```
Comment (HTRamsey) [inline src/Comms/USBBoardInfo.json:65]: ,

**Review by mrpollo — APPROVED** [review]
Vendor ID: 0x3643
Product ID: 0x15E0

https://github.com/PX4/PX4-Autopilot/blob/main/boards/zeroone/x6/firmware.prototype

**Comment by HTRamsey** [comment]
@mrpollo FYI this won't work because it is missing a comma. I am going to add a unit test so that we can know when it fails to pass

**Comment by mrpollo** [comment]
Thanks for reporting https://github.com/mavlink/qgroundcontrol/pull/12113

**Pull request merged**